js/src/hooks/useAcceptedCustomerDataTerms.test.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+import { renderHook } from '@testing-library/react-hooks';
+
+/**
+ * Internal dependencies
+ */
+import { useAppDispatch } from '.~/data';
+import useAcceptedCustomerDataTerms from './useAcceptedCustomerDataTerms';
+
+describe( 'useAcceptedCustomerDataTerms', () => {
+	it( 'Returns the correct status when set to true', () => {
+		const { result } = renderHook( () => {
+			const { receiveAcceptedTerms } = useAppDispatch();
+			receiveAcceptedTerms( { status: true } );
+
+			return useAcceptedCustomerDataTerms();
+		} );
+
+		expect( result.current.acceptedCustomerDataTerms ).toBeTruthy();
```

Comment:
If it expects exactly a `true` value, using `toBe( true )` would be a bit nicer.

```suggestion
		expect( result.current.acceptedCustomerDataTerms ).toBe( true );
```

Reply:
👍

Reply:
👍